scripts/pi-hole/js/messages.js

```diff
@@ -57,6 +57,24 @@ function renderMessage(data, type, row) {
         "</pre> to get the group configuration for this client."
       );
 
+    case "HOSTNAME":
+      var hint = "";
+      for (var i = 0; i < row.blob2 + row.message.length + 2; i++) hint += " ";
```

Comment:
Not sure this is readable, better break it to a new line

Reply:
`xo` asked me to do it this way. I had it in two lines, initially

Reply:
You need brackets.

Reply:
I will replace this by a shorter method.

Reply:
```suggestion
      var hint = new Array(row.blob2 + row.message.length + 3).join(" ");
```